cudax/include/cuda/experimental/__cuco/detail/hash_functions/utils.cuh

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2023-2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef _CUDAX__CUCO_DETAIL_HASH_FUNCTIONS_UTILS_CUH
+#define _CUDAX__CUCO_DETAIL_HASH_FUNCTIONS_UTILS_CUH
+
+#include <cuda/__cccl_config>
+#include <cuda/std/cstddef>
+
+#if defined(_CCCL_IMPLICIT_SYSTEM_HEADER_GCC)
+#  pragma GCC system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_CLANG)
+#  pragma clang system_header
+#elif defined(_CCCL_IMPLICIT_SYSTEM_HEADER_MSVC)
+#  pragma system_header
+#endif // no system header
+
+#include <cuda/std/__cccl/prologue.h>
+
+namespace cuda::experimental::cuco::detail
+{
+
+template <typename T, typename U, typename Extent>
+constexpr _CCCL_HOST_DEVICE T load_chunk(U const* const data, Extent index) noexcept
+{
+  auto const bytes = reinterpret_cast<::cuda::std::byte const*>(data);
+  T chunk;
+  memcpy(&chunk, bytes + index * sizeof(T), sizeof(T));
```

Comment:
All function calls need to be fully qualified
```suggestion
  _CUDA_VSTD::memcpy(&__chunk, __bytes + __index * sizeof(T), sizeof(T));
```
Is there a reason this cannot be just a `_CUDA_VSTD::bit_cast<T>` ?

Reply:
`load_chunk` helper function load's 4 byte chunks from the input key `Key`, which can be of any size. So, I think it's not possible to use `bit_cast` here.